source/compatibility.txt

```diff
@@ -26,14 +26,14 @@ MongoDB v4.0 or later.
 
 .. note::
 
-   We recommend that you only use the {+mkc+} with an official version
+   We recommend that you only use the {+connector+} with an official version
    of the MongoDB server. We cannot guarantee it functions correctly
    with any other version.
 
 Kafka Compatibility
 -------------------
 
-The {+mkc+} requires Kafka Connect v2.1.0 or later. It's
+The {+connector+} requires Kafka Connect v2.1.0 or later. It's
```

Comment:
Suggestion:
Use the source constant for Kafka Connect since the source constant loses some usefulness when not used in all instances. Similar comment for "Apache Kafka" on the next lines.

Possibly beyond the scope of the ticket, but should be a quick and not-debatable fix. To limit the scope of this PR, I would address only the ones on this page for "Apache Kafka" and not every instance of every source constant.

Reply:
Replaced on compatibility, whats new, and introduction.